packages/assets-controllers/src/NftController.ts

```diff
@@ -425,15 +430,17 @@ export class NftController extends BaseController<NftConfig, NftState> {
         return await this.getNftInformationFromApi(contractAddress, tokenId);
       });
     }
-    return {
-      ...openSeaMetadata,
-      name: blockchainMetadata.name ?? openSeaMetadata?.name ?? null,
-      description:
-        blockchainMetadata.description ?? openSeaMetadata?.description ?? null,
-      image: blockchainMetadata.image ?? openSeaMetadata?.image ?? null,
-      standard:
-        blockchainMetadata.standard ?? openSeaMetadata?.standard ?? null,
-    };
+
+    return Object.assign(
+      {
+        name: null,
+        description: null,
+        image: null,
+        standard: null,
+      },
+      openSeaMetadata,
```

Comment:
Do these need ...?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
Oh true I forgot that. Thanks!